cocos/rendering/custom/framework.ts

```diff
@@ -66,13 +66,18 @@ export function defaultWindowResize (ppl: BasicPipeline, window: RenderWindow, w
     ppl.addDepthStencil(`ShadowDepth${id}`, Format.DEPTH_STENCIL, shadowSize.x, shadowSize.y);
 }
 
+const _resizedWindows: RenderWindow[] = [];
+
 export function dispatchResizeEvents (cameras: Camera[], builder: PipelineBuilder, ppl: BasicPipeline): void {
     if (!builder.windowResize) {
         // No game window resize handler defined.
         // Following old prodecure, do nothing
         return;
     }
 
+    // Resize all windows.
+    // Notice: A window might be resized multiple times with different cameras.
+    // User should avoid resource collision between different cameras.
```

Comment:
What will happen if two or more cameras have have the same size?

Reply:
The custom pipeline resize logic of the specific render window will be called multiple times with different cameras.

This is by design, in order to:
1. Reduce memory usage to reuse resources of the render window.
2. Share render targets between different cameras.
User can also use camera name/index, to avoid collision if they really want.

Reply:
The cameras' sizes are the same of the render window.